config/rbac/role.yaml

```diff
@@ -210,6 +210,12 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
```

Comment:
could you reference also adding networkpolicies RBAC in the PR description

Reply:
Done